ENH: Add aarch64 and ppc64le

larsoner opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] ~~Reset the build number to `0` (if the version changed)~~
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Need `glad2-cmake` on aarch64 for https://github.com/conda-forge/vtk-feedstock/pull/368. Figured I'd try adding `ppc64le` while I was in there.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ No valid build backend found for Python recipe for package `glad2` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.
 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of `<output 0` output, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of `<output 0` output, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of `<output 0` output, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).
 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13314701724](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13314701724). Examine the logs at this URL for more detail.</sub>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

larsoner commented:
Okay all green, fixed parsing errors, updated to [CFEP-25](https://github.com/conda-forge/cfep/blob/main/cfep-25.md), and (what I originally meant to do!) added ppc64-le and aarch64 variants of `glad2-cmake`.

Ready for review/merge from my end @cNoNim @traversaro !
